chain/market/fundmanager.go

```diff
@@ -289,123 +309,166 @@ func (a *fundedAddress) saveState() {
 	}
 }
 
-// The result of processing the request queues
+// The result of processing the reservation / release queues
 type processResult struct {
+	// Requests that completed without adding funds
+	cancelled []*fundRequest
```

Comment:
nit: I find the use of the term "cancel" confusing. as I understand it, what this actually means is: I don't actually need to send a chain message for this reserve request.

It doesn't mean (the first two things that came to my head):
- cancel an add funds in progress (not sure why I thought this cause there's no cancel for chain messages)
- cancel a reserve request (not the case -- the funds are reserved, they just don't need a chain message)

One suggestion: "covered" -- I covered this request without having to add funds.

Reply:
Good call, I will change the name 👍